libraries/matrixui/src/main/kotlin/io/element/android/libraries/matrix/ui/media/ImageLoaderFactories.kt

```diff
@@ -33,11 +33,17 @@ class DefaultLoggedInImageLoaderFactory @Inject constructor(
     override fun newImageLoader(matrixClient: MatrixClient): ImageLoader {
         return ImageLoader
             .Builder(context)
-            .okHttpClient { okHttpClient.get() }
             .components {
+                add(
+                    OkHttpNetworkFetcherFactory(
+                        callFactory = {
+                            okHttpClient.get()
```

Comment:
This does not seem to be used, not sure what's happening here.

Reply:
I set a breakpoint on `okHttpClient.get()` and starts the application in debug mode (from AS). Images are loaded but the code never breaks.

Reply:
This sounds wrong then :/ 
According to the documentation, should be enough

Reply:
I have added logs and confirm that this code is not used. Still digging

Reply:
Actually this is fine. The `OkHttpClient` is not used to load images from the homeserver (avatars, images in room, etc.), we use the SDK API to retrieve them 🤦‍♂️.

The `OkHttpClient` is only used to load images for the map (location sharing), and in this case, the `CallFactory` is correctly invoked.